app/controllers/admin/users_controller.rb

```diff
@@ -1,8 +1,11 @@
-class Admin::UsersController < AdminController
+class Admin::UsersController < Admin::AdminController
   before_action :validates_role, only: [:update_status, :update_role, :destroy]
 
   def index
     @users = User.except_admins
```

Comment:
What is happening here, 
Now if you are using search you need to make this part of the search criteria for sphinx, instead of mysql.

Reply:
i clicked the wrong line :)